docs/cog_guides/cog_manager_ui.rst

```diff
@@ -0,0 +1,244 @@
+.. _cogmanagerui:
+
+==============
+Cog Manager UI
+==============
+
+This is the cog guide for the core cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: This cog is not like the other cogs. It is loaded by default, not
+    included in the cogs paths and it cannot be unloaded. It contains needed
+    commands for cogs management.
+
+.. _cogmanaerui-usage:
+
+-----
+Usage
+-----
+
+This cog allows you to manage your cogs and where you can install them. Unlike
+V2, which had a ``cogs`` folder where everything was installed, you can
+install V3 cogs everywhere, and also make them cross-compatible with other
+instances!
+
+The most basic command is :ref:`paths <cogmanagerui-command-paths>`, which
+will list you all of the currently set paths.
+
+You can add a path by using the :ref:`addpath <cogmanagerui-command-addpath>`
+command. All cogs in that path will be available for the bot and listed in
+the :ref:`cogs <cogmanagerui-command-cogs>`. You can then load
+or unload them.
+
+.. TODO add ref to load and unload commands
+
+.. _cogmanagerui-usage-installation:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+How to install a local package without using downloader
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Let's suppose you made a cog request on the `cog board <https://cogboard.red>`_
+and now you want to add your own cog to Red. You should have a folder that
+looks like this:
+
+.. image:: ../.resources/cog_manager_ui/custom-cog-example.png
+
+You will first need to add a cog path to your instance. For that, use the
+:ref:`addpath <cogmanagerui-command-addpath>` command with a new directory.
+
+Create a folder somewhere (should stay accessible) and copy its path. A path
+looks like this:
+
+*   Windows: ``C:\Users\username\Documents\MyCogs``
+*   macOS: ``/Users/username/Documents/MyCogs``
+*   Linux: ``/home/username/Documents/MyCogs``
+
+You can now use the command we talked about before: type ``[p]addpath
+<your_path>``.
+
+.. attention:: A path shouldn't have spaces in it. If it does, add quotation
+    marks, or an antislash before the space.
+
+In that ``MyCogs`` folder, you can drop your package. You should now have
+something like this:
+
+.. image:: ../.resources/cog_manager_ui/cog-path.png
+
+Now if you type ``[p]cogs``, your new cog should be listed, and you will be
+able to load it!
+
+.. _cogmanagerui-commands:
+
+--------
+Commands
+--------
+
+.. note:: The whole cog is locked to the
+    :ref:`bot owner <getting-started-permissions>`. If you are not the owner
+    of the instance, you can ignore this.
+
+.. _cogmanagerui-command-cogs:
+
+^^^^
+cogs
+^^^^
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]cogs
+
+**Description**
+
+Return a list of loaded and unloaded cogs on the bot.
+
+Cogs are unloaded by default, this is where you can find your cogs if you
```

Comment:
```suggestion
Cogs are unloaded by default. This is where you can find your cogs if you
```
I don't feel like these are (relatively) closely related so probably best to separate them with a full stop.